form.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
```

Comment:
With Jekyll, you don't have to copy-paste heading stuff like this into every page -- it's a lot simpler to manage the website if e.g. updating the header only involves updating one file!

So you can get rid of lines 1-99 in this file and simply replace it with:
```
---
title: Get involved
layout: page
---
```

Then Jekyll knows to automatically use the `page` layout (see the file in `_layouts/page.html`)!

Reply:
Amended my code!